src/AppInstallerCLICore/Argument.cpp

```diff
@@ -91,4 +94,14 @@ namespace AppInstaller::CLI
         args.push_back(ForType(Args::Type::RetroStyle));
         args.push_back(ForType(Args::Type::VerboseLogs));
     }
+
+    void Argument::OutputEnableMessage(Execution::Context& context) const

```

Comment:
>OutputEnableMessage [](start = 19, length = 19)

This should not be here, at least as a direct output function.  If I'm guessing at its use, then it should generated a string to be part of the CommandException generated by parsing.